docs/content-services/components/search-date-range-tabbed.component.md

```diff
@@ -1,16 +1,15 @@
 ---
-Title: Search date range advanced tabbed component
+Title: Search date range tabbed component
 Added: v6.2.0
 Status: Active
 Last reviewed: 2023-07-10
```

Comment:
Hmm what we have approach for editing documentations? Should we update for all updated md files "Last reviewed" field to let everyone knows that there is new version of documentation for this component? If yes then please apply for every md file.

@MichalKinas @DenysVuika what is proper approach - should we update this date or not?

Reply:
I think that if major change happens or component replaces another one we should update this date

Reply:
ok so I guess we don't need update it right @MichalKinas? Then I'm closing this comment, please reopen comment in case you think we should update it